doc/development-guide.md

```diff
@@ -69,12 +69,12 @@ If you are planning on making a large set of changes or a major architectural ch
 **Review Process:**
 Once your PR has has been submitted for review the following critieria will need to be met before it will be merged:
 * Each PR needs reviews accepting the change from at least two developers for merging
-  * It is common to request reviews from those reviewers automatically suggested by github
+  * It is common to request reviews from those reviewers automatically suggested by GitHub
```

Comment:
The title of PR needs to be more accurate and concise. Titles such as `Updating quick user guide` don't give a any useful information to reviewers. This applies to commit messages as well.